[READY FOR REVIEW] Brazilian Portuguese - Introduction to Web Accessibility

adielcristo opened the pull request:
[Direct link to preview](https://deploy-preview-924--wai-website.netlify.app/fundamentals/accessibility-intro/pt-br)

Translate resource pages/fundamentals/introduction/index.md to pt-BR

Resolves https://github.com/w3c/wai-translations/issues/253

## To do
- [x] Add @israelcefrin to the contributors list. See https://github.com/w3c/wai-website/pull/924#issuecomment-2546086103

israelcefrin commented:
Hello @adielcristo and @remibetin, I’ll be reviewing this translation PR and plan to complete it by December 9, 2024.

israelcefrin commented:
Hello @adielcristo 

I have completed my review and I am pleased to say that there are only a few items I would like to address. Most of these relate more to formal rules than to the translation itself.
You will find my review below, along with my suggestions. Please feel free to review them and adjust the text if you agree with my comments.

## Alert box:
1) Missing accentuation: word "ingles" should be "inglês"

## Related resource/recurso relacionado:
1) Yet the video has subtitles, it would be helpful if letting the users know that the audio is in English. It may be achieved including this information in the link label. E.g.: Vídeo Introdução à acessibilidade web e Padrões W3C (4 minutos, vídeo em inglês).

## Accessibilidade em Contexto
1) The term "*Web*" is usually written with capital "W". In Brazilian Portuguese we also noted it as in English. E.g. https://www.w3.org/Translations/WCAG20-pt-br/ . I would recommend an extended replace to keep the uniformity with other W3C documents throughout the whole document.

2) Adjusting the verbal mode from "indicativo" to "subjuntivo" (a wishfull thinking, not yet a reality but it is possible to become true): 
 pessoas desenvolvedoras e organizações que _desejam_ => pessoas desenvolvedoras e organizações que _desejem_


## O que é acessibilidade web
1)  Noun "óculos" is plural and requires that adjectives or verbs flex with it:  um braço quebrado ou óculos _perdido_ => um braço quebrado ou óculos _perdidos_ . Another option would be changing it to: um braço quebrado ou _um par_ de óculos perdido. 

2) Vídeo Perspectivas da acessibilidade web (YouTube) => Vídeo Perspectivas da acessibilidade web (YouTube, _vídeo em inglês_)

### Mais informações sobre o que é acessibilidade
1) The noun "_inglês_"(English) doesn't require initial capital letter in Brazilian Portuguese. Also, the official WCAG Brazilian Portuguese translation uses "inglês": https://www.w3.org/Translations/WCAG20-pt-br/
Recommended to replace every "em Inglês" by "em inglês" 

## Acessibilidade é importante para pessoas, negócios e a sociedade
1) The link to CRPD should include a label to let users know that the link target content is in English: 
sobre os Direitos das Pessoas com Deficiência (UN CRPD _texto em inglês_).

2) "Mídia impressa" is connected mostly to newspaper and it doesn't relate to other print material like journals or books. I think that we should change it to include any print media. Also "visual media" is a single term to any form of communication or expression that relies on visual elements, e.g.: television, photography, Art, etc... The noun "media" in this sentence relates only to "visual". Hence, I suggest to reword the statement from:
as barreiras de acessibilidade à mídia impressa, de áudio e visual
To:
as barreiras de acessibilidade _a materiais impressos, áudio e mídia visual_...

Also, removing the "crase" from the letter "a" because it relates to more than one element(plural) and it becomes a preposition instead of "preposition + article"

3) The structure fro the following sentence:
"o design acessível melhora a experiência geral da pessoa usuária e a satisfação"

Could be improved if we keep both criteria together (experiência e satisfação)
"o design acessível melhora a experiência geral e a satisfação da pessoa usuária .."

4) Missing "em inglês" warning in the link label of " Fatores legais e de política.". It should be:
...  Fatores legais e de política(em inglês).  ". 


### Tornando seu site acessível
- Como atender às WCAG (Referência rápida) => Como atender às WCAG (Referência rápida, em inglês)

## Exemplos
###Texto alternativo para imagens
1) Missing the info about link target content in English:
	-  As imagens devem incluir texto alternativo equivalente => As imagens devem incluir texto alternativo equivalente (_em inglês_)

2) The example image is still showing an alt text in English. Will it be image translated?

### Entrada de teclado
1) Missing the info about link target content in English:
	- ele disponibiliza todas as funcionalidades a partir de um teclado => ele disponibiliza todas as funcionalidades a partir de um teclado (em inglês).

### Transcrições para áudio
1) Missing the info about link target content in English:
	- Também existem serviços de transcrição que => Também existem serviços de transcrição(em inglês) que

remibetin commented:
Thanks for this careful review @israelcefrin! 

For the most part, I let @adielcristo consider your remarks, but I would like to provide some helpful context related to some of yourcomments. 

> Missing the info about link target content in English

"(in English)" is automatically added to links targeting internal pages without a Brazilian Portuguese translation. 

We know the existing system could be improved. Please feel free to add your input in the related issue: https://github.com/w3c/wai-translations/issues/100

In the meantime, we ask translators to not manually add "(in English)" to other links, even for external links. See [How to Translate a WAI Resource – Links](https://www.w3.org/WAI/about/translating/resources/technical-steps/#links)

> Will it be image translated?

For now, this image is not ready for translation and needs to remain in English in the translations.

> The noun "inglês"(English) doesn't require initial capital letter in Brazilian Portuguese. Also, the official WCAG Brazilian Portuguese translation uses "inglês": https://www.w3.org/Translations/WCAG20-pt-br/
Recommended to replace every "em Inglês" by "em inglês"

If you come to an agreement with @adielcristo on this, I suggest either of you open a pull request in `wai-website-data` repository, to update the [`translations.yml` file - line 358](https://github.com/w3c/wai-website-data/blob/b9d6043d68e2b81ee4fbb0abc72a0e25f974c272/translations.yml#L358).

adielcristo commented:
Hi @israelcefrin, thanks for reviewing this translation. Here are my considerations:

> ## Alert box:
> 
>     1. Missing accentuation: word "ingles" should be "inglês"

This text is translated in the [wai-website-data//translations.yml](https://github.com/w3c/wai-website-data/blob/master/translations.yml) file. I'll send a PR updating it.

> ## Accessibilidade em Contexto
> 
>     1. The term "_Web_" is usually written with capital "W". In Brazilian Portuguese we also noted it as in English. E.g. https://www.w3.org/Translations/WCAG20-pt-br/ . I would recommend an extended replace to keep the uniformity with other W3C documents throughout the whole document.

This one is a little tricky. Even on the latest WCAG translation, excluding the WGAC's name itself, the terms "Web" and "web" are "evenly" used. I opted for the lowercase version as it's being used by some large organizations such as the [Federal Senate](https://www12.senado.leg.br/manualdecomunicacao/estilos/web). I could not find a unique rule that could be used, except maybe when we are referring to (The World Wide) Web vs. some web. Also, when used as an adjective, lowercase makes more sense. @remibetin Thoughts on this?

>     2. Adjusting the verbal mode from "indicativo" to "subjuntivo" (a wishfull thinking, not yet a reality but it is possible to become true):
>        pessoas desenvolvedoras e organizações que _desejam_ => pessoas desenvolvedoras e organizações que _desejem_

Fixed.

> ## O que é acessibilidade web
> 
>     1. Noun "óculos" is plural and requires that adjectives or verbs flex with it:  um braço quebrado ou óculos _perdido_ => um braço quebrado ou óculos _perdidos_ . Another option would be changing it to: um braço quebrado ou _um par_ de óculos perdido.

Fixed.

>     2. "Mídia impressa" is connected mostly to newspaper and it doesn't relate to other print material like journals or books. I think that we should change it to include any print media. Also "visual media" is a single term to any form of communication or expression that relies on visual elements, e.g.: television, photography, Art, etc... The noun "media" in this sentence relates only to "visual". Hence, I suggest to reword the statement from:
>        as barreiras de acessibilidade à mídia impressa, de áudio e visual
>        To:
>        as barreiras de acessibilidade _a materiais impressos, áudio e mídia visual_...

Fixed.
 
> Also, removing the "crase" from the letter "a" because it relates to more than one element(plural) and it becomes a preposition instead of "preposition + article"

Fixed.

>     3. The structure fro the following sentence:
>        "o design acessível melhora a experiência geral da pessoa usuária e a satisfação"
> 
> 
> Could be improved if we keep both criteria together (experiência e satisfação) "o design acessível melhora a experiência geral e a satisfação da pessoa usuária .."

Fixed.

@remibetin already addressed the changes regarding the target resource language and image translation.

remibetin commented:
> This one is a little tricky. Even on the latest WCAG translation, excluding the WGAC's name itself, the terms "Web" and "web" are "evenly" used. I opted for the lowercase version as it's being used by some large organizations such as the [Federal Senate](https://www12.senado.leg.br/manualdecomunicacao/estilos/web). I could not find a unique rule that could be used, except maybe when we are referring to (The World Wide) Web vs. some web. Also, when used as an adjective, lowercase makes more sense. @remibetin Thoughts on this?

This is a little tricky indeed. I have no expertise in Brazilian Portuguese so I defer to your judgement on this.

In English, [W3C Manual of Style](https://www.w3.org/Guide/manual-of-style/) requests the following:

> **Web (on its own)**: always capitalize
> **Web (as part of a phrase)**: either capitalize or lower case "Web" (e.g., Web developer or web developer, Web project or web project, Web page or web page, Web application or web application

For _Web (on its own)_: I assume capitalization is best in most languages. 
For _Web (as part of a phrase)_: unless you can think of a well-established practice in Brazilian Portuguese language, I recommend that you choose for this resource whatever approach seems best and consistent to you at the moment.

Then, please [open a "Discuss a glossary term" issue](https://github.com/w3c/wai-translations/issues/new/choose) to continue discussing what would be the best option to apply in Brazilian Portuguese translations of WAI resources. The resulting wording could then be included in a Brazilian Portuguese glossary.

Existing glossaries are available at https://github.com/w3c/wai-translations/tree/main/glossaries

adielcristo commented:
> > This one is a little tricky. Even on the latest WCAG translation, excluding the WGAC's name itself, the terms "Web" and "web" are "evenly" used. I opted for the lowercase version as it's being used by some large organizations such as the [Federal Senate](https://www12.senado.leg.br/manualdecomunicacao/estilos/web). I could not find a unique rule that could be used, except maybe when we are referring to (The World Wide) Web vs. some web. Also, when used as an adjective, lowercase makes more sense. @remibetin Thoughts on this?
> 
> This is a little tricky indeed. I have no expertise in Brazilian Portuguese so I defer to your judgement on this.
> 
> In English, [W3C Manual of Style](https://www.w3.org/Guide/manual-of-style/) requests the following:
> 
> > **Web (on its own)**: always capitalize
> > **Web (as part of a phrase)**: either capitalize or lower case "Web" (e.g., Web developer or web developer, Web project or web project, Web page or web page, Web application or web application
> 
> For _Web (on its own)_: I assume capitalization is best in most languages. For _Web (as part of a phrase)_: unless you can think of a well-established practice in Brazilian Portuguese language, I recommend that you choose for this resource whatever approach seems best and consistent to you at the moment.
> 
> Then, please [open a "Discuss a glossary term" issue](https://github.com/w3c/wai-translations/issues/new/choose) to continue discussing what would be the best option to apply in Brazilian Portuguese translations of WAI resources. The resulting wording could then be included in a Brazilian Portuguese glossary.
> 
> Existing glossaries are available at https://github.com/w3c/wai-translations/tree/main/glossaries

@remibetin I reviewed the usage of Web/web. I'll keep the uppercase only in the proper names and open a discussion about this glossary term. Thanks.

remibetin commented:
> I reviewed the usage of Web/web. I'll keep the uppercase only in the proper names and open a discussion about this glossary term. Thanks.

Thanks @adielcristo.

@israelcefrin What do you think? Feel free to also provide your input in [issue #270](https://github.com/w3c/wai-translations/issues/270).

israelcefrin commented:
Hi both,

I agree to @adiel since we have some words in Brazilian Portuguese that come from English, e.g. site, website, etc... that don't require italicizing them anymore. We can treat "web" as a native noun which means applying the same grammar rules for the native ones.

remibetin commented:
Thanks @israelcefrin. Considering your contribution to the review, I think right to make your name appear as "Contributor" to the translation. Let me know how you would like to see your name written.

@adielcristo @israelcefrin Is there anything left for you to discuss before I do a final review and publish the translation?

remibetin commented:
> Let me know how you would like to see your name written.

@israelcefrin For information, the related policy is at https://www.w3.org/WAI/about/translating/#links

adielcristo commented:
> @adielcristo @israelcefrin Is there anything left for you to discuss before I do a final review and publish the translation?

I’m good.

remibetin commented on pages/fundamentals/introduction/index.pt-BR.md:
```diff
@@ -0,0 +1,242 @@
+
+A acessibilidade web depende de vários componentes trabalhando juntos, incluindo tecnologias web, navegadores web e outros \"agentes da pessoa usuária\", ferramentas de autoria e sites.
+
+A Iniciativa de Acessibilidade Web ([WAI](/about/participating/)) do W3C desenvolve especificações técnicas, diretrizes, técnicas e recursos de suporte que descrevem soluções de acessibilidade. Estes são considerados padrões internacionais para acessibilidade web; por exemplo, as <abbr title="Diretrizes de Acessibilidade de Conteúdo Web (WCAG)">WCAG</abbr> 2.0 também são um padrão <abbr title="Or ...
```
```suggestion
A Iniciativa de Acessibilidade Web ([WAI](/about/participating/)) do W3C desenvolve especificações técnicas, diretrizes, técnicas e recursos de suporte que descrevem soluções de acessibilidade. Estes são considerados padrões internacionais para acessibilidade web; por exemplo, as <abbr title="Diretrizes de Acessibilidade para Conteúdo Web">WCAG</abbr> 2.0 também são um padrão <abbr title="Organização Internacional para Padronização">ISO</abbr>: ISO/<abbr title="Comissão Eletrotécnica Internacional">IEC</abbr> 40500.
```

"Diretrizes de Acessibilidade para Conteúdo Web" is used in the [Authorized Translation of WCAG 2.0](https://www.w3.org/Translations/WCAG20-pt-br/) and the [Candidate Authorized Translation of WCAG 2.2](https://www.w3c.br/traducoes/wcag/wcag22-pt-BR/)

adielcristo commented on pages/fundamentals/introduction/index.pt-BR.md:
```diff
@@ -0,0 +1,242 @@
+{:/}
+
+-   Para uma introdução aos requisitos de acessibilidade e padrões internacionais, consulte [[Princípios de acessibilidade]](/fundamentals/accessibility-principles/).
+-   Para entender algumas barreiras comuns de acessibilidade da perspectiva de testes, consulte [[Verificações fáceis – Uma primeira revisão]](/test-evaluate/preliminary/).
```
> @adielcristo Could you complete the translation to make it clear the resource helps perform a first review "**of Web Accessibility**"?

Done.

remibetin commented:
Thanks @adielcristo. 

The only remaining step will be to add @israelcefrin to the contributors list. I will publish the translation right after.

remibetin commented:
> Thanks @israelcefrin. Considering your contribution to the review, I think right to make your name appear as "Contributor" to the translation. Let me know how you would like to see your name written.

> For information, the related policy is at https://www.w3.org/WAI/about/translating/#links

@israelcefrin Have you had a chance to read my previous comment? https://github.com/w3c/wai-website/pull/924#issuecomment-2546086103

I think we'll be able to publish the translation right after, ideally today.

israelcefrin commented:
> Thanks @israelcefrin. Considering your contribution to the review, I think right to make your name appear as "Contributor" to the translation. Let me know how you would like to see your name written.

Hello @remibetin , I have overlooked this question, my apologies.
Please, write my name as follows:
Israel Cefrin
